sys/newlib/Makefile.include

```diff
@@ -69,4 +69,4 @@ ifeq (1,$(USE_NEWLIB_NANO))
 endif
 
 # Newlib includes should go after GCC includes.
-export INCLUDES += $(NEWLIB_INCLUDES)
+export INCLUDES := $(NEWLIB_INCLUDES) $(INCLUDES)
```

Comment:
the comment above says, that order should be reversed? So either correct here or the comment, and explain w reasoning for certain order.